airflow/providers/amazon/aws/hooks/redshift_sql.py

```diff
@@ -44,6 +58,10 @@ class RedshiftSQLHook(DbApiHook):
     hook_name = "Amazon Redshift"
     supports_autocommit = True
 
+    def __init__(self, *args, aws_conn_id: str = "aws_default", **kwargs) -> None:
```

Comment:
`aws_conn_id` might be None, in this case default `boto3` strategy will use (usual EC2 Instance Profile or ECS Execution Role)